CMSgov/mint-app
# [MINT-2990] Update PBG Filer View

# MINT-2990

## Description

- Add several additional items to the PBG filter view

## How to test this change

1. Filter read view to PBG's filter view
2. In Operations, evaluation, and learning, Add IDDOC support and answer `Will you produce Benefit Enhancement Files?`
2. Add all payment types (Claims based, non-claims based, and Grants)
4. Verify that the PBG filter view has all that is listed in AC

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
